### PR TITLE
Complete

### PR DESCRIPTION
**Pull request by apellegrino**
I didn't get an email for creating my own repo so I just forked one. I think I overheard Gautam having this problem as well. Hopefully this works just as well.

### CODE REVIEW

File: ex1.jmd [hunk 1]
```diff
@@ -168,7 +168,9 @@ b_d64 = A_d64*x_d64
 maximum(Array(b_d64) .- b_h)
 ```
 
-Now, how do the results compare to your expectations?  
+Now, how do the results compare to your expectations?
+
```
Comment (eford) [inline ex1.jmd]: The different compilers for CPU and GPU and optimizing the order of operations differently.

**Comment by eford** [comment]
Good job.  This worked.  FYI- The link was item #4 in the Canvas announcement with subject "Serial code feedback on github, Lab 5 feedback on github & bug alert, Lab 7 avaliable: Astro 528"

**Comment by eford** [comment]
BTW- Did you finish lab 6?  I didn't notice a PR from you.

**Comment by apellegrino** [comment]
Not yet - I'm behind on the project so I haven't completed Ex. 3. I'll just push Exs. 1 and 2 now.

**Pull request closed without merging**